tests/linktype_name/test.yaml

```diff
@@ -0,0 +1,12 @@
+requires:
+  min-version: 8
+
```

Comment:
Is there not a pcap to reuse instead of putting a new one ?

Reply:
Good point; I'll make that adjustment.